composer.json

```diff
@@ -9,18 +9,44 @@
   ],
   "license": "MIT",
   "require": {
-    "symfony/console": "^2.8",
-    "symfony/filesystem": "^2.8",
-    "keboola/storage-api-client": "^8.0",
     "keboola/php-temp": "^0.1",
+    "keboola/storage-api-client": "^8.0",
     "keboola/symfony-console-nested-formatter-helper": "^0.1",
+    "symfony/console": "^2.8",
+    "symfony/filesystem": "^2.8"
+  },
+  "require-dev": {
+    "phpstan/phpstan-shim": "^0.9.2",
     "phpunit/phpunit": "^5.7",
+    "slevomat/coding-standard": "^4.4",
     "squizlabs/php_codesniffer": "^3.0"
   },
+  "scripts": {
+    "tests": "phpunit --debug",
+    "phpcs": "phpcs -n --ignore=vendor --extensions=php .",
```

Comment:
ještě by se dal přidat `phpcbf` (automaticky fixuje fixovatelné phpcs violations)  - stejné parametry jako phpcs a instaluje se rovnou s ním.

Reply:
no, ale jak to ty fixovatelne phpcs violations commitne? phpcbf na travisu imho nema smysl

Reply:
to nesouvisí s travisem - failne ti build tak si pustíš `composer phpcbf` a voila! Máš fixnuto co šlo.

Reply:
... lokálně.

Reply:
Zrovna třeba na přidání čárky na konec polí je to ideální.

Reply:
asi si to myslel tak ze se to jenom prida do scripts aby to clovek mohl jednoduseji spustit `composer phpcfb` misto copy paste nastaveni `phpcs` a prejmenovani na `phpcfb`? Tak jsem to delal doted:) takze s timhle souhlasim.

Reply:
ty carky na konci v tomhle PR jsem nefixoval rucne:) Rucne jsem delal asi jenom ty veci co zahlasil `phpstan`